src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariantsIntegrationTest.java

```diff
@@ -551,7 +566,7 @@ public void testKeepSelectionID() throws IOException {
     @Test
     public void testExcludeSelectionID() throws IOException {
         final String testFile = getToolTestDataDir() + "complexExample1.vcf";
-        final String idFile = getToolTestDataDir() + "complexExample1.vcf.id";
+        final String idFile = getToolTestDataDir() + "complexExample1.vcf.id.list";
```

Comment:
Can you add a test case with one or more literal rsIDs, since that's now possible?